### PR TITLE
Embeded tweets

### PR DESCRIPTION
**Pull request by bvi1994**
Added the twitter widget and made some of the buttons more uniform.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@ <h2>Code for San José</h2>
           <blockquote>
             <p>Where makers, designers, developers, and subject matter experts come together to use tech to solve civic problems.</p>
             <p><a class="btn btn-primary btn-lg" href="{{ "/docs/home/" | prepend: site.baseurl }}" role="button">Learn more</a></p>
-    </di      
+    </di
```
Comment (clarle) [inline index.html]: Typo here? Looks like invalid HTML.

File: index.html [hunk 2]
```diff
@@ -42,29 +42,40 @@ <h3 class="text-center">Create</h3>
...
+          <p>
+            Organizations interested in renting out the beautiful Bowers Institute space should contact <a href="mailto:abault@thetech.org">Alison Wilhelm</a>.
+          </p>
+      </div>
+      <div class="col-sm-3">
+        <a class="twitter-timeline"
+        href="https://twitter.com/codeforsanjose?ref_src=twsrc%5Etfw"
+        data-width="300"
+        data-height="400">
+        Tweets by codeforsanjose</a>
```
Comment (clarle) [inline index.html]: How does this look on a mobile device? The site right now is responsive, so would be good to do a quick check on a phone to make sure it doesn't break the layout there.

Comment (bvi1994) [reply index.html]: This is what I have when testing it in Responsive Mode - Apple iPhone SE

https://screenshots.firefox.com/kSsrtEMf0XirStWG/127.0.0.1

File: index.html (hunk 1 above)
Comment (bvi1994) [reply index.html]: I just removed it! Don't know how that happened.

**Comment by clarle** [comment]
LGTM :shipit:

**Pull request merged**